From: Jakub Jirutka <jakub@jirutka.cz>
Date: Mon, 29 Aug 2016 00:18:00 +0200
Subject: [PATCH] Fix compile error on aarch64

NativeRegisterContextLinux_arm64 uses user_pt_regs and user_fpsimd_state
which are defined in asm/ptrace.h.

--- a/source/Plugins/Process/Linux/NativeRegisterContextLinux_arm64.cpp
+++ b/source/Plugins/Process/Linux/NativeRegisterContextLinux_arm64.cpp
@@ -29,6 +29,7 @@
 
 // System includes - They have to be included after framework includes because they define some
 // macros which collide with variable names in other modules
+#include <asm/ptrace.h>
 #include <sys/socket.h>
 // NT_PRSTATUS and NT_FPREGSET definition
 #include <elf.h>
